processes/kneaddata.nf

```diff
@@ -12,17 +12,16 @@ process kneaddata {
     path human_genome
 
     output:
-    tuple val(sample), path("${sample}_kneaddata_paired_{1,2}.fastq.gz")
-    path "${sample}_kneaddata_unmatched_{1,2}.fastq.gz"
+    tuple val(sample), path("${sample}_kneaddata.fastq.gz")
     path "${sample}_kneaddata*.fastq.gz" , optional:true , emit: others
     path "${sample}_kneaddata.log"                       , emit: log
 
     shell:
-    
+
     """
     echo $sample
 
-    kneaddata --input ${reads[0]} --input ${reads[1]} \
+    kneaddata --input $reads \
```

Comment:
This isn't sufficient if we want to support both paired and single. Need to do something like you did in the metaphlan workflow to account for that.